core/src/database/PostgreSQLBackend.cpp

```diff
@@ -70,5 +79,15 @@ namespace ledger {
                                             soci::session &session) {
             throw make_exception(api::ErrorCode::IMPLEMENTATION_IS_MISSING, "Change of password is not handled with PostgreSQL backend.");
         }
+
+        void PostgreSQLBackend::initSSLLibraries() {
```

Comment:
You should wrap that function in another function adding the static initializer safety (or do it inside that function, I don’t really care). And we must ensure there’s a TLS on the actual initialization, otherwise it’s going to be madness when using two backends in two different threads.